src/app/shared/store/provider.actions.ts

```
@@ -94,51 +120,85 @@ export class GetProviderViewCompetitions {
+  static readonly type = '[user] send Draft for Moderation';
+
   constructor(public id: string) {}
 }
 
-export class CreateWorkshop {
-  static readonly type = '[provider] create Workshop';
-  constructor(public payload: Workshop) {}
+export class OnDraftSendForModerationSuccess {
+  static readonly type = '[provider] send for Moderation success';
+
+  constructor(public payload: void) {}
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)